### ISSUE TITLE
 When will the code be published?

### ISSUE THREAD

**Issue by gongyibei** [sajabdoli/Environmental_sound_classification#1]
When will the code be published?

**Comment by jacksongoode** [comment]
Also curious!

**Comment by MichelleYang2017** [comment]
I tried to reproduce this work but failed

**Comment by bluesky9606** [comment]
I tried to reproduce this work but failed，too.The accuracy of my reproduction is only about 64%, which is not accurate enough for the article

**Comment by MichelleYang2017** [comment]
me too 杨明雪 邮箱：michelleyang2017a@gmail.com 签名由 网易邮箱大师 定制 On 12/23/2020 14:17, bluesky9606 wrote: I tried to reproduce this work but failed，too.The accuracy of my reproduction is only about 64%, which is not accurate enough for the article — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by MichelleYang2017** [comment]
> I tried to reproduce this work but failed，too.The accuracy of my reproduction is only about 64%, which is not accurate enough for the article

The accuracy of my reproduction is only about 67%

**Comment by previtus** [comment]
I must admit, I am also unable to reproduce the paper's results. Noting the instructed folds defined by https://urbansounddataset.weebly.com/urbansound8k.html (and the advice not to shuffle the data in between these premade folds), my model's validation accuracy when training with folds 1-9 and testing on fold 10 is about 67%. I have not yet run a full 10 fold cross-validation though.
Furthermore, if I allow reshuffling of the whole dataset (basically ignoring the original folds), which is highly dis-encouraged (and even marked as wrong as it may lead to training samples from the same source bleeding into the test data), I am getting around 82-84%.

However, I might have made a mistake at some point (loading, pre-processing, etc), so I would like to know if anyone else was able to reproduce the results - or if the official code will be available?

_Edit: more about the official / unofficial data splits in https://arxiv.org/abs/2004.07301_
